maplibre_gl_platform_interface/lib/src/location.dart

```diff
@@ -84,6 +84,21 @@ class LatLngBounds {
     return <dynamic>[southwest.toJson(), northeast.toJson()];
   }
 
+  bool contains({required LatLng point}) {
+    final bool isLatitudeInBounds = point.latitude >= southwest.latitude &&
```

Comment:
```suggestion
    final isLatitudeInBounds = point.latitude >= southwest.latitude &&
```
The CI fails because of a lint in this line.
```console
lib/src/location.dart:88:11 - Unnecessary type annotation on a local variable. Try removing the type annotation. - omit_local_variable_types
```